### PR TITLE
Conformance results for distribution-spec/v1.0/openregistry

### PR DESCRIPTION
**Pull request by guacamole**
- This is the first PR from OpenRegistry to apply for OCI-distribution spec certification

Signed-off-by: guacamole <gunjanwalecha@gmail.com>

### CODE REVIEW

File: distribution-spec/v1.0/openregistry/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+### How to Run this project locally:
+OpenRegistry is not Go Gettable right now because of a dependency issue with Go-Skynet. To build this project locally, please use the following method:
+```bash
+git clone https://github.com/containerish/OpenRegistry.git
+make mod-fix
+go build
+```
+
+### Recreating the Results: 
+https://raw.githubusercontent.com/containerish/OpenRegistry/master/scripts/recreating.sh
```
Comment (jdolitsky) [inline distribution-spec/v1.0/openregistry/README.md:40]: Please put the steps for recreating directly in this document. Less general information about the project itself

For example please see https://github.com/opencontainers/oci-conformance/blob/main/distribution-spec/v1.0/zot/README.md

File: distribution-spec/live/badges.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+| Type | Status |
+|------|--------|
+| OCI - Push | [![OCI Push](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-push.yml/badge.svg)](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-push.yml)|
+| OCI - Pull | [![OCI Pull](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-pull.yml/badge.svg)](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-pull.yml)|
+| OCI - Content Management | [![OCI Content Management](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-content-management.yml/badge.svg)](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-content-management.yml)|
+| OCI - Content Discovery | [![OCI Content Discovery](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-content-discovery.yml/badge.svg)](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-content-discovery.yml)|
```
Comment (jdolitsky) [inline distribution-spec/live/badges.md:6]: Please put one badge per line (no additional markdown)

For example please see https://github.com/opencontainers/oci-conformance/blob/main/distribution-spec/live/zot/badges.md

Additionally, this file should be nested under the `$dir`. For example, `distribution-spec/live/openregistry/badges.md`

File: distribution-spec/v1.0/openregistry/PRODUCT.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+vendor: https://github.com/containerish
+name: OpenRegistry
+version: V0.0.2-beta
+website_url: https://openregistry.dev
+repo_url: https://github.com/containerish/OpenRegistry
+documentation_url: https://blog.openregistry.dev
+product_logo_url: https://avatars.githubusercontent.com/u/90045228?s=200&v=4
```
Comment (jdolitsky) [inline distribution-spec/v1.0/openregistry/PRODUCT.yaml:7]: Please remove querystring from image (`?s=200&v=4`)

File: distribution-spec/v1.0/openregistry/junit.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+<testsuite name="OCI Distribution Conformance Tests" tests="59" failures="0" errors="0" time="106.054">
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Populate registry with test blob" classname="OCI Distribution Conformance Tests" time="4.02818423"></testcase>
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Populate registry with test layer" classname="OCI Distribution Conformance Tests" time="6.576234994"></testcase>
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Populate registry with test manifest" classname="OCI Distribution Conformance Tests" time="4.784856368"></testcase>
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Get the name of a tag" classname="OCI Distribution Conformance Tests" time="5.121218909"></testcase>
+    <testcase name="OCI Distribution Conformance Tests Pull Setup Get tag name from environment" classname="OCI Distribution Conformance Tests" time="0.000542962">
+        <skipped>/Users/guacamole/oss/distribution-spec/conformance/01_pull_test.go:82&#xA;you have skipped this test.&#xA;/Users/guacamole/oss/distribution-spec/conformance/setup.go:344</skipped>
```
Comment (jdolitsky) [inline distribution-spec/v1.0/openregistry/junit.xml:8]: If you want, you can fuzz out the username used in this file (up to you either way)

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: distribution-spec/v1.0/openregistry/README.md (hunk 1 above)
Comment (guacamole) [reply distribution-spec/v1.0/openregistry/README.md:40]: updated!

File: distribution-spec/live/badges.md (hunk 2 above)
Comment (guacamole) [reply distribution-spec/live/badges.md:6]: Moved the file inside openregistry/ and removed extra markdown

File: distribution-spec/v1.0/openregistry/PRODUCT.yaml (hunk 3 above)
Comment (guacamole) [reply distribution-spec/v1.0/openregistry/PRODUCT.yaml]: removed the querying from logo url

File: distribution-spec/v1.0/openregistry/junit.xml (hunk 4 above)
Comment (guacamole) [reply distribution-spec/v1.0/openregistry/junit.xml]: removed the skipping cased which removed the username thingy

File: distribution-spec/live/openregistry/badges.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+[![OCI Push](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-push.yml/badge.svg)](https://github.com/containerish/openregistry/actions/workflows/oci-dist-spec-push.yml)
+
```
Comment (jdolitsky) [inline distribution-spec/live/openregistry/badges.md:2]: Please remove these extra lines between badges

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: distribution-spec/v1.0/openregistry/junit.xml (hunk 4 above)
Comment (jdolitsky) [reply distribution-spec/v1.0/openregistry/junit.xml]: Please revert this change.. The test XML itself should not be modified. Just meant to say that you could replace "guacamole" in the output if you wanted to

File: distribution-spec/v1.0/openregistry/PRODUCT.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+vendor: https://github.com/containerish
```
Comment (jdolitsky) [inline distribution-spec/v1.0/openregistry/PRODUCT.yaml:1]: Can this be "Containerish" ? (vs. URL)

**Pull request merged**